resources/qml/delegates/TextMessage.qml

```diff
@@ -29,6 +29,11 @@ MatrixText {
         border-collapse: collapse;
         border: 1px solid " + Nheko.colors.text + ";
     }
+    blockquote {
+        margin: 0;
```

Comment:
Applying this only to blockquotes is a bit weird, if there is a paragraph following the quote. Maybe fix those cases as well or don't set this completely to 0?

Reply:
In every `<blockquote>` is exactly one `<p>`, according to my testing. The paragraph following the quote has it's own margins which is the same as the margin between to `<p>`. (See last image)

Reply:
Having no margins between the `<p>`s inside the blockquote, but margins outside seems weird to me. I would expect those both to have approximately the same margins.

Reply:
Ok, removed all `margin: 0;`s. It'll look like this:

![screenshot_2021-09-04T17:44:09](https://user-images.githubusercontent.com/3681516/132100369-7bb2521f-2deb-48bb-a9b2-fa25980904aa.png)

Reply:
Is that still okay for you or do you want to reduce the vertical height more by maybe shrinking those margins by half?

Reply:
It doesn't really matter to me, since I don't think I ever encountered multiple blockquote-levels or multiple blockquotes of the same level in a row before.

Reply:
Then this looks good to me, thank you!